templates/includes/npm.php

```diff
@@ -161,7 +161,7 @@
     <a href="npm.php">Install the <code>enable-a11y</code> NPM project</a>.
   </li>
   <li>
-    You can import the module using require like this:
+    You can import the module using requirements like this:
```

Comment:
We are talking about the actual `require` code. So maybe it should be like the following?

```suggestion
    You can import the module using <code>require</code> like this:
```